app/javascript/widget/views/Home.vue

```diff
@@ -163,6 +170,10 @@ export default {
         this.showAttachmentError = false;
       }, 3000);
     });
+    bus.$on(BUS_EVENTS.START_NEW_CONVERSATION, () => {
+      this.isOnCollapsedView = true;
+      this.startNewConversation = true;
```

Comment:
Can we have a better naming for this variable? `startNewConversation` seems like an action rather than a state which holds the value of whether the pre-chat form is enabled or not.

Reply:
How about `ON_NEW_CONVERSATION`?

Reply:
`isOnNewConversation` sounds good.